spring-batch-core/src/test/java/org/springframework/batch/core/explore/support/MapJobExplorerFactoryBeanTests.java

```diff
@@ -36,8 +39,12 @@
 	public void testCreateExplorer() throws Exception {
 
 		MapJobRepositoryFactoryBean repositoryFactory = new MapJobRepositoryFactoryBean();
-		repositoryFactory.getObject().createJobExecution("foo", new JobParameters());
-		
+		JobExecution jobExecution = repositoryFactory.getObject().createJobExecution("foo", new JobParameters());
+
+		//simulating a running job execution
+		jobExecution.setStartTime(new Date());
+		repositoryFactory.getObject().update(jobExecution);

```

Comment:
The two calls to `repositoryFactory.getObject();` in this method can be extracted to a variable.